### DISCUSSION TITLE
Receber e-mails de pessoas pelo meu site

### DISCUSSION THREAD

**Discussion by igorpinhh** [frontendbr/forum#1069] [Discussão]
Pessoal qual a forma mais eficiente de permitir que pessoas me enviem e-mails diretamente do meu site? qual seria a melhor forma de fazer isso em vossas opiniões? penso em fazer usando Ruby e recentemente fiquei sabendo que é possível com Node.
Aceito sugestões, dicas, qualquer coisa. 
Obrigado!

**Comment by mahenrique94** [comment]
Depende, hoje seu site está feito como? Tem que ver se utilizou alguma linguagemde back-end.

Sua necessidade trata-se de um formulário para contato, o mesmo pode ser feito de N maneiras, o primeiro ponto é saber as tecnologias utilizadas no site atualmente.

Esse recurso pode ser feito em Ruby, Node, PHP, Java, Kotlin, etc...

**Comment by didiraja** [comment]
A mais simples é com PHP. Tanto com a função mail quanto usando o PHP Mailer. Mas como o @mahenrique94 disse, depende de como seu back-end está ou será feito.

**Comment by bizarro** [comment]
Se você não estiver afim de usar backend para isso, tem uns serviços tipo o [Formspree](https://formspree.io/), que é gratuíto desde que você use um action que envia um email com template deles. 😅

**Comment by luizbills** [comment]
A [netlify](https://www.netlify.com/) também oferece serviço de enviar emails sem backend. Além de uma ótima hospedagem gratuita.

**Comment by pietrobelluno** [comment]
Assim como o @luizbills eu recomendo o [https://www.netlify.com/](url) se teu site for estatico. Deploy do site em 30 segundos, integracao com git e o formulario tu consegue manipular a resposta, pode fazer em ajax para nao precisar mudar de tela, é muito bem documentado e muito simples de usar, e o melhor de tudo é que é gratuito hahahah. Eu estou usando ele atualmente e achei muito sensacional, vale a pena testar

**Comment by ninetails** [comment]
Precisa ser mesmo email?

Existe Google Forms.

Em outro extremo, tem ferramentas também de gerenciamento de feedbacks de usuários, como o UserVoice.

Chat bots...

**Comment by igorpinhh** [comment]
@mahenrique94 estou usando node pois estou aprendendo ainda, mas nada mt complicado.

**Comment by igorpinhh** [comment]
@ninetails não necessariamente, posso usar isso, não me dei conta que podia usar o google forms, eu não manjo nada de back, sou do front...

**Comment by igorpinhh** [comment]
@luizbills obrigado, vou pesquisar sobre

**Comment by igorpinhh** [comment]
@lhbzr obrigadooooo.

**Comment by gerardo-junior** [comment]
recomendo pegar algum script php de envio por api ou usar alguma serviço tipo: https://www.mailgun.com/, https://www.mailjet.com/email-api/, https://sendgrid.com/solutions/email-api/ dependendo do que você queria, talvez criptografar seja uma boa nesses casos
